src/components/Forms/TextFieldHint.js

```diff
@@ -15,9 +15,11 @@ const styles = {
     transition: 'opacity 250ms cubic-bezier(0.23, 1, 0.32, 1) 0ms',
     pointerEvents: 'none',
     zIndex: 1,
+    visibility: 'hidden',
```

Comment:
Just to double check here, is this specifically so that screen readers won't read this content?

Reply:
Right. When you tab around in the screen reader and the hint isn't showing, the hint will still receive focus by the screen reader and read off

Reply:
Also, this will interrupt the CSS transition since it happens immediately. You want to use multiple transitions, one on visibility and one on opacity.  The visibility transition should have a delay value equal to the duration of the opacity transition.

Reply:
Wouldn't that be ok since the opacity will still be transitioning while visibility is already set? From testing, it looks like opacity transition still takes effect regardless of visibility.

Reply:
Before it would both fade in and fade out. After this change it will only fade in. Setting visible false happens immediately.